pkg/acsengine/azureconst.go

```diff
@@ -6,7 +6,9 @@ import "fmt"
 
 const (
 	// AzureProdFQDNFormat specifies the format for a prod dns name
-	AzureProdFQDNFormat = "%s.%s.cloudapp.azure.com"
+	AzurePublicProdFQDNFormat = "%s.%s.cloudapp.azure.com"

```

Comment:
It looks like these changes were done manually.

Instead, the PowerShell script needs to be updated so that it can output these things. We really need to make that thing not a PowerShell script...